mino/minogrpc/server.go

```diff
@@ -549,9 +550,35 @@ func (o *overlay) Join(addr, token string, certHash []byte) error {
 }
 
 func (o *overlay) makeCertificate() error {
+	// We take a substring from [1, N) because
+	// o.myAddrStr[0] == 'F' || o.myAddrStr[1] == 'O'
+	hostname, _, err := net.SplitHostPort(o.myAddrStr[1:])
+	if err != nil {
+		if _, ok := o.myAddr.(fake.Address); ok {
```

Comment:
I'm not really fond of this check specifically. Another alternative is to have fake.Address.String() return `127.0.0.1` but that might be confusing. Whatsay?